aml-development/frontend
# fix(menu): remove 'Themes' option for production environment

AMLOS-778

**Description**
The theme option in the menu is primarily for Developers. Put an option in place to remove that menu option when in a Production environment.

**Acceptance Criteria**
Configuration has an option to show/hide menu from theme
Production configuration hides theme by default
Development configuration shows theme by default

**How to Test**
Pull this branch

Start the frontend using `npm run prod`.  Verify that the Theme selector is not in the menu.  If you have SSL certs setup and can login that way, verify that the user can still change their theme by editing their profile

Start the frontend using `npm run dev`.  Verify that the Theme selector is in the menu and changes the site theme as expected.
